packages/web/src/pages/sign-up-page/pages/PickHandlePage.tsx

```diff
@@ -278,92 +240,78 @@ export const PickHandlePage = () => {
   )
 
   const initialValues = {
-    handle: value || ''
+    handle
   }
 
   return (
-    <Box h='100%' className={styles.outerContainer}>
-      <Formik
-        innerRef={formikRef}
-        initialValues={initialValues}
-        validationSchema={validationSchema}
-        onSubmit={handleSubmit}
-        validateOnChange={false}
-      >
-        {({ isSubmitting, isValid, isValidating }) => (
-          <Form>
-            <Flex direction='column' justifyContent='space-between' h='100%'>
-              <Flex
-                pv='3xl'
-                className={styles.contentContainer}
-                direction='column'
-                gap='xl'
+    <Formik
+      innerRef={formikRef}
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+      onSubmit={handleSubmit}
+      validateOnChange={false}
+    >
+      {({ isSubmitting, isValid, isValidating }) => (
+        <Flex as={Form} direction='column' h='100%'>
+          <Flex
+            direction='column'
+            gap='2xl'
+            alignSelf='center'
+            pv='xl'
+            ph={isMobile ? 'l' : 'xl'}
+            css={!isMobile && { maxWidth: 610 }}
+          >
+            <Flex gap={isMobile ? 's' : 'l'} direction='column'>
+              {isMobile ? null : (
+                <Text size='s' variant='label' color='subdued'>
+                  1 {messages.outOf} 2
```

Comment:
could make the messages.outOf a function that takes the numerator and denominator

Reply:
agreed!